client/src/App.tsx

```diff
@@ -4,13 +4,18 @@ import { Router } from '@reach/router'
 import Index from './pages/Index'
 import NotFound from './pages/NotFound'
 import Game from './pages/Game'
+import Lobby from './pages/Lobby'
+import axios from 'axios'
 
 const App: React.FC = () => {
+  axios.defaults.baseURL = 'https://localhost:5001'
```

Comment:
no need to use different origin, configure the proxy in `package.json` to point to `http://localhost:5000`